helm/ako/values.yaml

```diff
@@ -50,6 +50,7 @@ L7Settings:
 L4Settings:
   advancedL4: "false" # Use this knob to control the settings for the services API usage. Default to not using services APIs: https://github.com/kubernetes-sigs/service-apis
   defaultDomain: "" # If multiple sub-domains are configured in the cloud, use this knob to set the default sub-domain to use for L4 VSes.
+  autoFQDN: "<svc>.<ns>.<subdomain>" # ENUM: <svc>-<ns>.<subdomain>, <svc>.<ns>.<subdomain>, "false" If the value is false then the FQDN generation is disabled.
```

Comment:
This can have an empty value too, which will be equivalent to default ?

Reply:
yeah any value is in fact default. We are just specifying a `false` as way of being symbolic of the feature being turned off.